src/tools/rest-frontend/public/assets/translations/en.json

```diff
@@ -0,0 +1,370 @@
+    
+    "APP.NAV.BUTTON.LOGIN": "Login",
+    "APP.NAV.BUTTON.LOGOUT": "Logout",
+    "APP.NAV.BUTTON.MAIN.CONVERSION": "Convert snippet",
+    "APP.NAV.BUTTON.MAIN.DOCUMENTATION": "Documentation",
+    "APP.NAV.BUTTON.MAIN.HOME": "Home",
+    "APP.NAV.BUTTON.MAIN.INTRODUCTION": "Getting started",
+    "APP.NAV.BUTTON.MAIN.NEWS": "News",
+    "APP.NAV.BUTTON.MAIN.TUTORIALS": "Tutorials",
+    "APP.NAV.BUTTON.MAIN.USERS": "Users",
+    "APP.NAV.BUTTON.NEWNETRY": "New snippet",
+    "APP.NAV.BUTTON.ACCOUNT": "Account",
+    
+    "APP.USERS.DETAILS.BUTTON.CHANGE_EMAIL": "Change email",
+    "APP.USERS.DETAILS.BUTTON.CHANGE_PASSWORD": "Change password",
+    "APP.USERS.DETAILS.BUTTON.CHANGE_RANK": "Change rank",
+    "APP.USERS.DETAILS.LABEL.CREATEDAT": "Created at:",
+    "APP.USERS.DETAILS.LABEL.EMAIL": "Email:",
+    "APP.USERS.DETAILS.LABEL.EMAILNEW": "New email:",
+    "APP.USERS.DETAILS.LABEL.EMAILNEWCONFIRM": "Confirm new email:",
+    "APP.USERS.DETAILS.LABEL.PASSWORDNEW": "New password:",
+    "APP.USERS.DETAILS.LABEL.PASSWORDNEWCONFIRM": "Confirm new password:",
+    "APP.USERS.DETAILS.LABEL.RANK": "Rank:",
+    "APP.USERS.DETAILS.LABEL.RANKNEW": "New rank:",
+    "APP.USERS.DETAILS.LABEL.RANK.10": "Normal user (10)",
+    "APP.USERS.DETAILS.LABEL.RANK.50": "Moderator (50)",
+    "APP.USERS.DETAILS.LABEL.RANK.100": "Administrator (100)",
+    "APP.USERS.DETAILS.LABEL.USERNAME": "Username:",
+    "APP.USERS.DETAILS.LABEL.VALIDATION.EMAILNEW.EMAIL": "You have to enter a valid email address.",
+    "APP.USERS.DETAILS.LABEL.VALIDATION.EMAILNEW.REQUIRED": "You have to enter a new email if you want to change it.",
+    "APP.USERS.DETAILS.LABEL.VALIDATION.PASSWORDNEW.MINLENGTH": "The new password must be at least eight characters long.",
+    "APP.USERS.DETAILS.LABEL.VALIDATION.PASSWORDNEW.PATTERN": "The new password must contain at least one lower-case letter, one upper-case letter and one digit. We recommend also using special characters as well.",
+    "APP.USERS.DETAILS.LABEL.VALIDATION.PASSWORDNEW.REQUIRED": "You have to enter a new password if you want to change it.",
+    "APP.USERS.DETAILS.LABEL.VALIDATION.PASSWORDNEWCONFIRM.EQUAL_INPUT": "Your confirmation has to match the first input.",
+    "APP.USERS.DETAILS.LABEL.VALIDATION.PASSWORDNEWCONFIRM.REQUIRED": "You have to confirm the new password.",
+    "APP.USERS.DETAILS.LABEL.VALIDATION.RANKNEW.REQUIRED": "You have to select a new rank if you want to change it.",
+    "APP.USERS.DETAILS.NOTIFICATION.HEADER": "Change account information",
+    "APP.USERS.DETAILS.NOTIFICATION.MESSAGE.NEED_AUTHENTICATION": "You have to be authenticated to perform this action.",
+    "APP.USERS.DETAILS.NOTIFICATION.MESSAGE.USER_INSUFFICIENT_PERMISSIONS": "You have insufficient permissions to performt his action.",
+    "APP.USERS.DETAILS.NOTIFICATION.MESSAGE.USER_NOT_FOUND": "The user account you are trying to change does not exist. It is possible that it has been deleted in the mean time.",
```

Comment:
Theoretically possible, but in general it is not good to use the same loca key in many places, as it might be a problem in future.